### ISSUE TITLE
Remove editor's note from accessibility spec

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2578]
Looks like we missed the note in the status section about working to meet 1.0 while 1.1 is in development: https://www.w3.org/TR/epub-a11y-11/#sotd

@iherman shouldn't editor's notes never make it into a REC? Could we get pubrules to flag their presence?

**Comment by iherman** [comment]
That is essentially an erratum due to our own stupidity :-) This should have been removed when going to Rec.
